etc/kayobe/environments/aufn-ceph/configure-openstack.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+BASE_PATH=$1
+VENV_DIR=$BASE_PATH/venvs/ansible
+cd $BASE_PATH/src/
+[[ -d openstack-config ]] || git clone https://github.com/stackhpc/openstack-config-multinode.git -b generic-network openstack-config
```

Comment:
Note: Remember to change checkout branch once [this PR](https://github.com/stackhpc/openstack-config-multinode/pull/2) is merged.